back/app/policies/user_policy.rb

```diff
@@ -57,11 +57,11 @@ def destroy?
   end
 
   def block?
-    index?
+    index_xlsx?
   end
 
   def unblock?
-    index?
+    index_xlsx?
```

Comment:
nit picking but I'd have a slight preference for using `user&.active? && user.admin?` directly here if it's not referencing one of the 'default' methods (index, show etc)